okhttp/src/main/kotlin/okhttp3/internal/ws/RealWebSocket.kt

```diff
@@ -64,6 +64,7 @@ class RealWebSocket(
   private var extensions: WebSocketExtensions?,
   /** If compression is negotiated, outbound messages of this size and larger will be compressed. */
   private var minimumDeflateSize: Long,
+  private val closeTimeoutMillis: Long = CANCEL_AFTER_CLOSE_MILLIS,
```

Comment:
Not sure we need the default here.

Reply:
I am not sure whether RealWebSocket is considered public API so I did not want to change it unnecessarily

Reply:
It's not, but this would be changing this API regardless of the default, unless we used JvmOverloads.

Reply:
Yeah, you're reight. I removed it.